certified-connectors/App Power Forms/apiProperties.json

```diff
@@ -0,0 +1,39 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key (use e-mail to receive a trial key)",
+          "description": "The API Key for this api. You can use your e-mail address to receive a trial key.",
+          "tooltip": "Provide your API Key (use e-mail to receive a trial key)",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#6DD47E",
+    "capabilities": [], 
+    "policyTemplateInstances": [
+      {
+        "templateId": "pollingtrigger",
+        "title": "PollingTriggerPolicy_69bf0411-f6e0-cc7e-e7d1-5b504c8e9669",
```

Comment:
@TerhoAntila No secret-look-a-like values are allowed in the swagger due to Microsoft security policy.

Reply:
Will fix...

Reply:
Resolution committed.